packages/zoe/test/unitTests/zcf/zcfTesterContract.js

```diff
@@ -7,7 +7,9 @@ import '../../../exported';
  * @type {ContractStartFn}
  */
 const start = zcf => {
-  return { creatorFacet: zcf };
+  // make the `zcf` available to the tests
+  zcf.setTestJig();
```

Comment:
this call could be removed if you did an automatic `testJig.setZCF(zcf)` as described above

Reply:
Aye. It's fine for it to be explicit, but I can imagine making it always available. That's easy to change in an upwards compatible fashion, so worth discussion even after landing.  Mostly it will depend on how much we use the jig feature.